hash2curve/Cargo.toml

```
@@ -0,0 +1,25 @@
+[package]
+name = "hash2curve"
+version = "0.2.0-rc.0"
```

Perhaps this to keep the version number in sync with all of the other elliptic curve crates?

```suggestion
version = "0.14.0-rc.0"
```

---

Done.